packages/front-end/src/app/pages/labs/[labId]/seqera-run/[seqeraRunId].vue

```diff
@@ -11,89 +11,78 @@
 
   const labId = $route.params.labId as string;
   const seqeraRunId = $route.params.seqeraRunId as string;
+
   const seqeraRunReports = ref([]);
   const s3Contents = ref<S3Response>(null);
   const tabIndex = ref(0);
+  const runId = ref(''); // Run ID from query params
```

Comment:
can this be replaced with a `computed` and get rid of the state handling and the `watch` below?

Reply:
Good call!